torch/testing/_internal/common_fsdp.py

```diff
@@ -111,15 +111,18 @@ def _assert_module_states(
 def _zero_model(
     model: nn.Module,
     zero_buffers: bool = False,
+    summon_full=True,
 ):
     """Zeros the parameters and optionally buffers of ``model`` in place."""
-    for param in model.parameters():
-        with torch.no_grad():
-            param.zero_()
-    if zero_buffers:
-        for buffer in model.buffers():
+    ctx = FSDP.summon_full_params(model) if summon_full else suppress()
```

Comment:
Should we include any to-do or issue for following up on this? Or, could you remind me what the current status on this is?

Reply:
yeah, let me file an issue summarizing it.